mm2src/mm2_tests.rs

```diff
@@ -7183,7 +7228,7 @@ fn test_verify_message_utxo() {
       "coin":"RICK",
       "message":"test",
       "signature": "HzetbqVj9gnUOznon9bvE61qRlmjH5R+rNgkxu8uyce3UBbOu+2aGh7r/GGSVFGZjRnaYC60hdwtdirTKLb7bE4=",
-      "address":"R9o9xTocqr6CeEDGDH6mEYpwLoMz6jNjMW"
+      "address":"rck1qqk4t2dppvmu9jja0z7nan0h464n5gve8h7nhay"
```

Comment:
I meant the following test case in https://github.com/KomodoPlatform/atomicDEX-API/pull/1256#discussion_r862781655, which fails now, but should succeed:
```rust
let rc = block_on(mm_bob.rpc(&json! ({
    "userpass": mm_bob.userpass,
    "method":"verify_message",
    "mmrpc":"2.0",
    "id": 0,
    "params":{
      "coin":"RICK",
      "message":"test",
      "signature": "HzetbqVj9gnUOznon9bvE61qRlmjH5R+rNgkxu8uyce3UBbOu+2aGh7r/GGSVFGZjRnaYC60hdwtdirTKLb7bE4=",
      "address":"R9o9xTocqr6CeEDGDH6mEYpwLoMz6jNjMW"

    }
    })))
    .unwrap();

    assert!(rc.0.is_success(), "!verify_message: {}", rc.1);

    let response: Json = json::from_str(&rc.1).unwrap();
    let is_valid = &response["result"]["is_valid"];
    assert_eq!(is_valid, true);
```
Even if coin is activated in Segwit mode, it should verify the signature for legacy addresses too.